### ISSUE TITLE
It should not evaluate the permitted attributes of the input element that the type attribute value is dynamic.

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#263]
```tsx
<input type={type} placeholder="..." />
```

Results `The "placeholder" attribute is not allowed (invalid-attr)`

## Benchmark cross-reference

_Sync'd from `tests/external/snapshots/` (submodule=1429313, nu-validator@sha256:210bbc353b23, markuplint=5.0.0-rc.4, generated=2026-08-12 06:31:53 UTC)._

**Not directly verifiable against the current nu-validator fixture suite.** Reason: No fixture in the nu-validator suite probes dynamic `input[type]` attribute evaluation.

Fixtures could be authored manually under a separate tree, or contributed upstream to [`validator/validator`](https://github.com/validator/validator). See the [`bench-triage` skill](https://github.com/markuplint/markuplint/blob/dev/.claude/skills/bench-triage/SKILL.md) for the audit workflow.

**Comment by YusukeHirao** [comment]
I thought to need some interpretations for the spec of elements:

A. The attribute is empty.
B. Union types that all patterns of the value of the attribute.
C. Intersection types that all patterns of the value of the attribute.

If the element is the input and the attribute is the type:

A: `<input type={any}/>` is `<input type=""/>`
B: Allow only common attributes allowed on the element.
C: Allow all attributes allowed on the element.

And it must consider the permitted content and WAI-ARIA specs too.

It's able to change them by `specOptions` on the configuration.

**Comment by YusukeHirao** [comment]
However, It defines conditions for attributes as CSS Selector. It can make union types and intersection types if it uses the parser, but I think it is too complicated, and I need a long-time.

So I defer to resolving this issue.
I will think about it again after I release v2.

**Comment by YusukeHirao** [comment]
Verified against current `dev` HEAD — no longer reproduces:

```jsx
const X = ({ type }) => <input type={type} placeholder="..." />;
```

```
$ markuplint --config '{"parser":{"\\.jsx$":"@markuplint/jsx-parser"},"extends":["markuplint:html-standard"]}' input.jsx
✔ 1 file checked, 0 problems
```

Root cause and fix: `isValidAttr()` in `packages/@markuplint/rules/src/helpers.ts` resolves an attribute's `condition` (a positive-list selector keyed on another attribute's literal value, e.g. `input[type=text]`) via `attrMatches()`. When the condition references an attribute whose value is dynamic (JSX `type={type}`, Vue `:type`), the condition result is indeterminate — the old code still evaluated it against the literal (often empty/placeholder) attribute value and reported "disallowed" as a false positive.

Fixed by commit f04557d2f (`fix(rules): skip condition-disallow when a referenced attr value is dynamic`, merged in PR #3904, 2026-06-07): `conditionDependsOnDynamicAttr()` now extracts the attribute names referenced by the condition selector and skips the condition check entirely when any of them is dynamic on the element (`helpers.ts:33-50, 128-135`).

Closing as resolved. Removed the stale `secondary` bench-xref mapping for this Issue in the same commit as #460 (both no longer applicable).
